timestamp.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+workspacedir=$(head -n1 ../../README | cut -d ' ' -f 2)/$1
+
+echo "`date` and old time was `cat $workspacedir/$2`" > $workspacedir/$2
+
+ln -s $workspacedir/$2 $2
```

Comment:
this is the key, we'll have to create symlinks to disk for all generated tsc outputs. I'm assuming we can figure out the outputs based on the source files? Typical bazel problem there, and bazel provided typescript rules has to solve that already.

Reply:
> I'm assuming we can figure out the outputs based on the source files

Yep for sure. See [this](https://github.com/sconover/rules_multi_tsc/blob/master/rules/private/tsc.bzl#L45) code block.